src/scrconf/kde4_kdmrc.scr

```diff
@@ -0,0 +1,23 @@
+/**
+ * File:	kde4_kdmrc.scr
+ * Summary:	Agent for reading/writing kdmrc
+ * Author:	Ancor González Sosa <ancor@suse.de>
+ * Access:	read / write
+ *
+ * Read/Sets the values defined in
+ * <tt>/usr/share/kde4/config/kdm/kdmrc</tt>
+ */
+.kde4.kdmrc
+
+`ag_ini(
+  `IniAgent(
+    "/usr/share/kde4/config/kdm/kdmrc",
```

Comment:
I also found it smelly, but it was stated in the bug report as the proper place. I will ask some KDE packager just to be sure.